sdk/storage/inc/datalake/response_models.hpp

```diff
@@ -0,0 +1,168 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#pragma once
+
+#include "blobs/internal/protocol/blob_rest_client.hpp"
+#include "protocol/datalake_rest_client.hpp"
+
+namespace Azure { namespace Storage { namespace Files { namespace DataLake {
+
+  // ServiceClient models:
+
+  using ListFileSystemsResult = ServiceListFileSystemsResponse;
```

Comment:
I still oppose typedefs here.

The source of this kind of problems is because of defect in swagger. You're trying to work around it rather than fixing it.

Reply:
There is no point fixing it now since we are going to have the swagger generator rewritten anyway. This is not a breaking change to our customers when we switch to a swagger generator that resolves this issue in the future.